pytest-{{cookiecutter.plugin_name}}/appveyor.yml

```diff
@@ -0,0 +1,41 @@
+# What Python version is installed where:
+# http://www.appveyor.com/docs/installed-software#python
+
+environment:
+  matrix:
+    - PYTHON: "C:\\Python27"
```

Comment:
There's no advantage in using a build matrix in AppVeyor because it doesn't run parallel builds for open source projects... in fact using a build matrix when all you want is to execute `tox` tends to take longer to execute a build.